demos/python_demos/whiteboard_inpainting_demo/README.md

```diff
@@ -0,0 +1,76 @@
+# Whiteboard inpainting demo
+
+This demo demonstrates how to disable a person from a whiteboard and show
+text on the whiteboard using OpenVINO<sup>TM</sup>.
+
+## How it works
+
+The demo expects one of the next models in the Intermediate Representation (IR) format:
+
+    * Instance segmentation model
+    * Semantic segmentation model
+
+It can be your own models or pre-trained model from OpenVINO Open Model Zoo.
+In the `models.lst` are the list of appropriate models for this demo that can
```

Comment:
@snosov1 do we still have technical writers' reviews?

Reply:
Alina should take a look